flask_mongoengine/wtf/models.py

```diff
@@ -4,12 +4,12 @@
 class ModelForm(FlaskForm):
     """A WTForms mongoengine model form"""
 
-    def __init__(self, formdata=None, obj=None, **kwargs):
-        self.instance = (kwargs.pop('instance', None) or obj)
+    def __init__(self, formdata=None, **kwargs):
+        self.instance = (kwargs.pop('instance', None) or kwargs.get('obj'))
```

Comment:
Hmmm... `dict.get` defaults to `None`, right?

    assert {'a':1, 'b':2}.get('d') is None

(This is inconsistent with `getattr` and I was already struck by it once recently.)

Thanks anyway for taking the time to review your own code again. I appreciate.

And considering the little we dedicate to the maintenance of flask-mongoengine, a PR is appreciated, even for a trivial change.